packages/ui/src/components/Tabs/Tabs.tsx

```diff
@@ -8,52 +8,98 @@ const Root = React.forwardRef<
   React.ElementRef<typeof TabsPrimitive.Root>,
   React.ComponentPropsWithoutRef<typeof TabsPrimitive.Root>
 >((props, forwardedRef) => (
-  <TabsPrimitive.Root ref={forwardedRef} {...props} />
+  <TabsPrimitive.Root
+    ref={forwardedRef}
+    data-h2-max-width="base(100%)"
+    {...props}
+  />
 ));
 
 const List = React.forwardRef<
   React.ElementRef<typeof TabsPrimitive.List>,
   React.ComponentPropsWithoutRef<typeof TabsPrimitive.List>
->((props, forwardedRef) => (
-  <TabsPrimitive.List ref={forwardedRef} {...props} />
+>(({ children, ...rest }, forwardedRef) => (
+  <TabsPrimitive.List
+    data-h2-max-width="base(100%)"
+    data-h2-display="base(flex)"
+    data-h2-gap="base(x.25)"
+    data-h2-overflow="base(auto visible)"
+    data-h2-width="base(100%)"
+    data-h2-padding="base(0 x1)"
+    data-h2-z-index="base(1)"
+    data-h2-overflow-x="base(auto)"
+    data-h2-overscroll-behavior-x="base(contain)"
+    data-h2-scroll-snap-align="base(start)"
+    data-h2-scroll-snap-type="base(x mandatory)"
+    data-h2-scrollbar-width="base:hover(none)"
+    ref={forwardedRef}
+    {...rest}
+  >
+    {children}
+  </TabsPrimitive.List>
 ));
 
 const Trigger = React.forwardRef<
   React.ElementRef<typeof TabsPrimitive.Trigger>,
   React.ComponentPropsWithoutRef<typeof TabsPrimitive.Trigger>
->((props, forwardedRef) => (
-  <TabsPrimitive.Trigger
-    className="Tabs__Trigger"
-    data-h2-background-color="base(white)"
-    data-h2-border="base(1px solid gray)"
-    data-h2-border-top="
-      base:selectors[[data-state='inactive']](x.5 solid gray)
-      base:selectors[[data-state='active']](x.5 solid primary)
-      base:selectors[[data-state='inactive']]:hover(x.5 solid gray.dark)
-      base:selectors[[data-state='active']]:hover(x.5 solid primary.dark)"
-    data-h2-border-bottom="base:selectors[[data-state='active']](1px solid white)"
-    data-h2-cursor="base(pointer)"
-    data-h2-padding="base(x.5, x1)"
-    data-h2-margin="base(0, x.5, 0, 0)"
-    data-h2-position="base(relative)"
-    data-h2-location="base(1px, auto, auto, auto)"
-    data-h2-radius="base(s, s, 0, 0)"
-    data-h2-transition="base(border 100ms ease)"
-    ref={forwardedRef}
-    {...props}
-  />
-));
+>(({ children, ...rest }, forwardedRef) => {
+  const handleFocus: React.FocusEventHandler<HTMLButtonElement> = (event) => {
+    event.currentTarget.scrollIntoView({
+      block: "center",
+    });
+  };
```

Comment:
Is the centring along the vertical axis intentional? 

https://recordit.co/s86dQvUTRi

Reply:
Oops, no. That was to get the mobile views to scroll if you were using a keyboard but it seems to be scrolling the document and not the parent 😬 .

I may need to look into getting that working. Or maybe it is not needed 🤔

Reply:
Alright, I believe I figured it out! Had to do some math but it seems to be working now 😅 

[fix mobile scroll logic](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8656/commits/50499cf7c9ab3cb05938e11a8c5f57d45127f2ed)